composer.json

```diff
@@ -5,7 +5,7 @@
     "homepage" : "https://github.com/evert/phpdoc-md",
     "license" : "MIT",
     "require" : {
-        "php" : ">=5.3.1",
+        "php" : ">=5.5.9",
```

Comment:
I don't really see a need to drop PHP 5.4 because currently it 'just works', and we're just making it a bit harder for people on those PHP versions to use it.

If we ran into PHP 5.4-specific bugs, or if we wanted to use PHP 5.5-specific feature like Generators, then I think it's a completely valid reason to make PHP 5.5.0 the minimum... but currently I don't really see a reason to.